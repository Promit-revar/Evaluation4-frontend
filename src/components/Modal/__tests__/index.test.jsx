import React from 'react';
import {asFragment, render, screen} from '@testing-library/react';
import ModalComponent from '../index';
describe('Modal', () => {
  it('should render correctly', () => {
    const {asFragment}= render(<ModalComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});