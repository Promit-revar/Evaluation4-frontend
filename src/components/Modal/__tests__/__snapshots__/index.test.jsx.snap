// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should render correctly 1`] = `
<DocumentFragment>
  <div
    class="modal"
  >
    <div
      class="modal-content"
    />
  </div>
</DocumentFragment>
`;
